### PR TITLE
GitLab pipeline values

### PR DESCRIPTION
**Pull request by devinmarieb**
# Description
- moves GL pipeline values to the adoc snippet
- add link to config ref to find GL values on the pipeline values page

# Reasons
This information can now be integrated into the normal flow of our docs rather than live on the independent GL page. The config ref (markdown) table needs work as the env_var tag doesn't seem to work anymore and the values will not break/wrap in their column. I attempted some other markdown formatting options and none of them would break the value into two lines. For this reason, the config ref has a link to a properly formatted (adoc) table on another page instead.

Markdown table in config ref that is not actually added to this PR:
table screen shot

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [ ] Break up walls of text by adding paragraph breaks.
- [ ] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [ ] Keep the title between 20 and 70 characters.
- [ ] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [ ] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [ ] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

File: jekyll/_includes/snippets/pipeline-values.md [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
+`pipeline.number`{:.env_var} | GitHub, Bitbucket | Integer | A project unique integer id for the pipeline
+`pipeline.project.git_url`{:.env_var} | GitHub, Bitbucket | String | The URL where the current project is hosted. For example, `https://github.com/circleci/circleci-docs`
+`pipeline.project.type`{:.env_var} | GitHub, Bitbucket | String | The lower-case name of the VCS provider, E.g. “github”, “bitbucket”.
+`pipeline.git.tag` | GitHub, Bitbucket | String | The name of the git tag that was pushed to trigger the pipeline. If the pipeline was not triggered by a tag, then this is the empty string.
+`pipeline.git.branch`{:.env_var} | GitHub, Bitbucket | String | The name of the git branch that was pushed to trigger the pipeline.
+`pipeline.git.revision`{:.env_var} | GitHub, Bitbucket | String | The long (40-character) git SHA that is being built.
+`pipeline.git.base_revision`{:.env_var} | GitHub, Bitbucket | String | The long (40-character) git SHA of the build prior to the one being built. **Note:** While in most cases  .
```
Comment (kbatuigas) [inline jekyll/_includes/snippets/pipeline-values.md:10]:
```suggestion
`pipeline.git.base_revision`{:.env_var} | GitHub, Bitbucket | String | The long (40-character) git SHA of the build prior to the one being built. **Note:** While in most cases `pipeline.git.base_revision` will be the SHA of the pipeline that ran before your currently running pipeline, there are some caveats. When the build is the first build for a branch, the variable will not be present. In addition, if the build was triggered via the API, the variable will not be present.
```

File: jekyll/_includes/snippets/pipeline-values.md [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+`pipeline.git.base_revision`{:.env_var} | GitHub, Bitbucket | String | Will be the SHA of the pipeline that ran before your currently running pipeline, there are some caveats. When the build is the first build for a branch, the variable will not be present. In addition, if the build was triggered via the API, the variable will not be present.
```
Comment (kbatuigas) [inline jekyll/_includes/snippets/pipeline-values.md:11]:
```suggestion

```
Duplicate line

**Review by kbatuigas — CHANGES_REQUESTED** [review]

File: jekyll/_includes/snippets/pipeline-values.md (hunk 2 above)
Comment (devinmarieb) [reply jekyll/_includes/snippets/pipeline-values.md:11]: Oh sorry, that's not actually a duplicate, but it's referencing itself in the description, so I think that's why it looks confusing. The env_var tag doesn't force a break on those anyway, so I can remove the second one so it's less confusing to look at.

**Review by kbatuigas — APPROVED** [review]
LGTM!

**Pull request merged**